.jenkins/actions/run_performance.sh

```diff
@@ -6,11 +6,14 @@ ROOT_DIR="$(dirname "$(dirname "$SCRIPTPATH")")"
 test -n "${experiment}" || exitError 1001 ${LINENO} "experiment is not defined"
 test -n "${backend}" || exitError 1002 ${LINENO} "backend is not defined"
 
+if [ "$experiment" = "c12_6ranks_standard" ]; then
+    data_path="/project/s1053/fv3core_serialized_test_data/7.0.0/c12_6ranks_standard"
+fi
 if [ "$experiment" = "c96_6ranks_baroclinic" ]; then
     data_path="/project/s1053/fv3core_serialized_test_data/7.2.3/c96_6ranks_baroclinic"
 fi
-if [ "$experiment" = "c12_6ranks_standard" ]; then
-    data_path="/project/s1053/fv3core_serialized_test_data/7.0.0/c12_6ranks_standard"
+if [ "$experiment" = "c128_6ranks_baroclinic" ]; then
+    data_path="/project/s1053/fv3core_serialized_test_data/7.2.3/c128_6ranks_baroclinic"
```

Comment:
The data folder, or the head location, /project/s1053/fv3core_serialized_test_data could be abstracted to a variable for reuse in this file.

Reply:
done